### DISCUSSION TITLE
Easy way for performance improvements - RefreshDriver Tick

### DISCUSSION THREAD

**Discussion by whitespacecode** [pixijs/pixijs#8715] [Q&A]
**Current setup:**
I'm using pixi to draw a static 2D (topview) plan with objects.
Once everything is drawn the user can click on these objects where some actions can happen on these objects.

**Problem**
When idle and doing nothing, the browser reports 25-30% GPU usage.
Most of it comes from the RefreshDriver Tick being called x times over and over even when nothing is happening.

**Actions**
I already set my PIXI.application to 1 fps which helps a little bit at idle state 16-20% GPU usage.
```
app.ticker.autoStart = false;
app.ticker.maxFPS = 1;
```

Another thing i could do is completely stop the ticker `app.ticker.stop();` But then nothing works anymore.
I'm guessing i have need to manually start the ticker every time the user interacts with the objects and stop it again when nothing happens? Maybe there is a better solution ?

**Comment by Ericlm** [comment]
I suggest you have a look at [pixi performance tips](https://gist.github.com/GoodBoyDigital/5aea403f8528fb89c4f7639d6a51353f), you'll certainly find some things to do!
